src/Tribe/RSVP.php

```diff
@@ -465,6 +458,15 @@ public function generate_tickets( ) {
 
 			$has_tickets = true;
 
+			/**
+			* RSVP specific action fired just before a RSVP-driven attendee ticket for an event is generated
+			*
+			* @param $event_id ID of event
+			* @param $ticket Ticket of product
+			* @param $_POST post paremeters comes from RSVP Form
```

Comment:
Should have noted this before also. True story: I have seen in our own code a case where we did this (or it could have been a different superglobal like `$_GET`) and it was written that way in the callback function, ie:

``` php
function fire_sometime( $_POST, ... ) {}
```

Simply to discourage others from that (it will trigger errors on newer php runtimes) how about naming the variable differently here in the docblock?